Assets/LoggingManager/LoggingManager.cs

```diff
@@ -74,45 +61,27 @@ void Awake()
         }
     }
 
-    public void GenerateUIDs()
+    public void SetSavePath(string path)
     {
-        sessionID = Md5Sum(System.DateTime.Now.ToString(SystemInfo.deviceUniqueIdentifier + "yyyy:MM:dd:HH:mm:ss.ffff").Replace(" ", "").Replace("/", "").Replace(":", ""));
-        deviceID = SystemInfo.deviceUniqueIdentifier;
+        this.savePath = path;
     }
 
-    public Dictionary<string, Dictionary<int, object>> GetLog(string collectionLabel)
+    public void GenerateUIDs()
     {
-        return new Dictionary<string, Dictionary<int, object>>(collections[collectionLabel].log);
+        sessionID = Md5Sum(System.DateTime.Now.ToString(SystemInfo.deviceUniqueIdentifier + "yyyy:MM:dd:HH:mm:ss.ffff").Replace(" ", "").Replace("/", "").Replace(":", ""));
+        deviceID = SystemInfo.deviceUniqueIdentifier;
     }
```

Comment:
`GenerateUIDs` is no longer needed right? you were using the GUID.newGuid function now, right?

Reply:
Yes, let's remove it.